src/inc/regdisp.h

```diff
@@ -359,6 +343,45 @@ inline LPVOID GetRegdisplayStackMark(REGDISPLAY *display) {
 
 #endif
 
+#if defined(_WIN64) || defined(_TARGET_ARM_) || (defined(_TARGET_X86_) && defined(WIN64EXCEPTIONS))
+// This needs to be implemented for platforms that have funclets.
+inline LPVOID GetRegdisplayReturnValue(REGDISPLAY *display)
+{
+    LIMITED_METHOD_CONTRACT;
+
+#if defined(_TARGET_AMD64_)
+    return (LPVOID)display->pCurrentContext->Rax;
+#elif defined(_TARGET_ARM64_)
+    return (LPVOID)display->pCurrentContext->X0;
+#elif defined(_TARGET_ARM_)
+    return (LPVOID)display->pCurrentContext->R0;
+#else
+    PORTABILITY_ASSERT("GetRegdisplayReturnValue NYI for this platform (Regdisp.h)");
+    return NULL;
+#endif
+}
+
+inline void SyncRegDisplayToCurrentContext(REGDISPLAY* pRD)
+{
+    LIMITED_METHOD_CONTRACT;
+
+#if defined(_WIN64)
```

Comment:
I would recommend changing the name of the stack pointer in the regdisplay for x86 from Esp to SP and then adding `|| defined(_TARGET_X86_)` to the `defined(_ARM_)` below.

It is kind of unfortunate that the REGDISPLAY for x86 and arm use DWORD as a type for the SP and ControlPC while x64 and arm64 use size_t. If that was not that way, the code here could be unified. But I guess that trying to make such change would have a ripple effect, so I would make it as a cleanup later.